tests/core/vm/test_sharding_vm.py

```diff
@@ -90,7 +93,8 @@ def test_sharding_apply_transaction(shard_chain_without_block_validation):  # no
         assert state_db.get_storage(CREATE2_contract_address, 0) == 1
 
 
-def test_CREATE2_deploy_contract_edge_cases(shard_chain_without_block_validation):  # noqa: F811
+@pytest.mark.xfail(reason="#281", strict=True)  # noqa: F811
```

Comment:
I'm gonna eat my words and say that this is fine now that I know about the `strict` paramter.  That's great.

Reply:
Oooh! it looks like we can enable this globally by default: by adding to our `pytest.ini`

```
[pytest]
xfail_strict=true
```

Reply:
And... it might be nice to add the `raises` parameter to any of these that you expect to hang around for a while so that we also know they are failing for the expected reason.

https://docs.pytest.org/en/latest/skipping.html#raises-parameter